doc/ChangeLog.md

```diff
@@ -19,6 +19,8 @@ All notable changes to the project are documented in this file.
   such bridges is to use a VLAN interface on top
 - A VLAN filtering bridge now validates that no IP address has been
   set.  Use a VLAN interface on top for that (see above)
+- Add support for static multicast filters, both MAC and IPv4 groups
+  are supported.
```

Comment:
Does the model not allow IPv6 groups?

Reply:
No

Reply:
> Does the model not allow IPv6 groups?

My mistake, i thought it didn't but:
`typedef ip-multicast-group-address {
    type union {
      type ipv4-multicast-group-address;
      type ipv6-multicast-group-address;
    }
    description
      "This type represents a version-neutral IP multicast group
       address.  The format of the textual representation implies
       the IP version.";
  }`